Testing/ContinuousIntegration/AzurePipelinesWindows.yml

```diff
@@ -11,8 +11,20 @@ jobs:
 - job: Windows
   timeoutInMinutes: 0
   cancelTimeoutInMinutes: 300
-  pool:
-    vmImage: 'vs2017-win2016'
+  matrix:
```

Comment:
Wasn't this change recently merged? Or something very like it? What is different here?

Reply:
I remove the erroneous commit related to AZP changes.